plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2393,6 +2395,14 @@ elementgui.tool.stays_in_grid_when_crafting=Does item stay in crafting grid when
 elementgui.tool.damaged_on_crafting=<html>Damage item instead on crafting<br><small>Make sure to enable "stay in crafting grid" and that item is damageable
 elementgui.tool.tooltip_tip=<html>Special information about the tool:<br><small>Use comma to separate entries
 elementgui.tool.needs_a_name=Tool needs a name
+elementgui.villager_profession.name=Profession name:
+elementgui.villager_profession.point_of_interest=Point of interest:
```

Comment:
This is not a clean naming for a field for beginners, target block or something similar is better

Reply:
Using another name than what MC literally for 150 million users is not the best name... very not sure about this. It would totally not understandable for everyone knowing a bit about MC (commands just as an example)

Reply:
It's done, I just changed it

Reply:
@Goldorion POI is general term and can be used for many purposes. Then it can be `target block (point of interest)`

Also don't expect average Minecraft user to know technical terms, you can check https://mcreator.net/forum for a big proof of this

Reply:
I said this mainly because Mojang added POI to the /locate command in 1.19.